sql/2019/11_PWA/11_04b.sql

```diff
@@ -0,0 +1,25 @@
+#standardSQL
+# 11_04b: Manifests that are not JSON parsable
+CREATE TEMPORARY FUNCTION getManifestProps(manifest STRING)
```

Comment:
Why not simplify this to
```sql
CREATE TEMPORARY FUNCTION canParseManifest(manifest STRING)
RETURNS BOOLEAN LANGUAGE js AS '''
try {
  JSON.parse(manifest);
  return true;
} catch (e) {
  return false;
}
'''
```

Reply:
Just reusing 11_04. Doesn't take up any more time/resources.